db/seeds.rb

```diff
@@ -60,8 +60,10 @@
 
   num_users.times do |i|
     fname = Faker::Name.unique.first_name
-    lname = Faker::Name.unique.last_name
-    name = [fname, lname].join(" ")
+    # Including "\\:/" to help with identifying local issues with
```

Comment:
Changing how we build the names.  Let's assume everyone in our tests likes to create a bit antics.